arviz/tests/helpers.py

```diff
@@ -28,6 +28,7 @@
 
 _log = logging.getLogger(__name__)
 
+needs_emcee3 = pytest.mark.skipif(emcee.__version__ < '3', reason="emcee3 required")
```

Comment:
Will this string comparison work? If so why not do a string on line 99 as well?

Reply:
I checked in ipython interpreter and it worked. I can change it to use the same method in both cases.